### ISSUE TITLE
WebView does not load local HTML file

### ISSUE THREAD

**Issue by dyphen12** [react-native-community/react-native-webview#630]
I'm attempting to load a local HTML file to webview and sometimes does not find the file and sometimes does not have access.

This is the error ERR_ACCESS_DENIED
or ERR_FILE_NOT_FOUND

This is the code of the app.js if it works:

`/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow
 */

import React, {Component} from 'react';

import { StyleSheet, Text, View, Button,Platform } from "react-native";

import { WebView } from 'react-native-webview';
import HTML_FILE from "./AdListing/index.html"
const isAndroid= Platform.OS==='android';

export default class App extends Component<Props> {
  constructor(props) {
    super(props);
    this.state = { val: 1 };
  }
  renderElement() {
    if (this.state.val === 1) {
      return (
        <WebView
          style={{flex: 1}}
          originWhitelist={['*']}
          source={{ uri: "https://aboutreact.com" }}
          style={{ marginTop: 20 }}
          javaScriptEnabled={true}
          domStorageEnabled={true}
        />
      );
    } else if (this.state.val === 2) {
      return (
        <WebView
          style={{flex: 1}}
          originWhitelist={['*']}
          source={{ html: '<h1>Hello</h1>' }}
          style={{ marginTop: 20 }}
          javaScriptEnabled={true}
          domStorageEnabled={true}
        />
      );
    } else {
      if(isAndroid){
        return (
            <WebView
              style={{flex: 1}}
              originWhitelist={['*']}
              source={{uri:'file:///android_assets/index.html'}}
              style={{ marginTop: 20 }}
              javaScriptEnabled={true}
              domStorageEnabled={true}
            />
        )
      }else{
        return(
          <WebView
            style={{flex: 1}}
            originWhitelist={['*']}
            source={HTML_FILE}
            style={{ marginTop: 20 }}
            javaScriptEnabled={true}
            domStorageEnabled={true}
          />
        );
      }
    }
  }
  render() {
    return (
      <View style={{flex: 1, marginTop:30, backgroundColor:'white'}}>
        <Button title="Load from URL" onPress={() => this.setState({ val: 1 })}/>
        <Button title="Load HTML Text" onPress={() => this.setState({ val: 2 })}/>
        <Button title="Load from Local HTML FIle" onPress={() => this.setState({ val: 3 })}/>
        <View style={{flex: 1}}>
          {this.renderElement()}
        </View>
      </View>
    );
  }
}`

I want to show the local html file in the project but I cannot or I don't know, please help! Thanks

**Environment:**
 - OS: Android
 - OS version: 9.0
 - react-native version: 16.8.3
 - react-native-webview version: 5.11.0

**Comment by StevenPurn** [comment]
I have also been having problems getting android to load local files (images & html in my case) in the webview. I am always getting the same error, ERR_ACCESS_DENIED. I've confirmed that the file paths are correct and the files exist by rendering the image in an <Image /> component. I've also confirmed it isn't a permissions issue as I've seen suggested in some other discussions. Everything seems to work fine on iOS. Would love to know if there is a bug preventing it from working as expected or if we're doing something incorrectly.
